### ISSUE TITLE
VPC & Subnet Tags not Being Applied with Terraform

### ISSUE THREAD

**Issue by smslattery** [localstack/localstack#3031]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description

When creating a vpc or subnet resource by running `terraform apply`, any tags associated with the resource are not being applied.

## Additional Notes

When creating a different resource (a simple s3 was used to compare), the tags are applied successfully.

When running `terraform apply` a second time, the error `Error: error waiting for EC2 VPC (vpc-fefd8ae2) IPv6 CIDR to become disassociated: couldn't find resource (2 retries)` occurs.

When running `terraform apply` a third time, the tags show up as expected.

## Expected behavior

Output from terraform apply shows tags applied successfully:
```
Outputs:

vpc_id = vpc-fefd8ae2
vpc_tags = {
  "name" = "some-vpc"
}
```

## Actual behavior

Output from terraform apply shows empty map:
```
Outputs:

vpc_id = vpc-fefd8ae2
vpc_tags = {}
```

# Steps to reproduce

## Command used to start LocalStack

`IMAGE_NAME=localstack/localstack:0.11.5 localstack start`

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

This was executed using terraform v0.12.6

```
$ terraform apply -auto-approve
aws_vpc.vpc: Creating...
aws_vpc.vpc: Creation complete after 0s [id=vpc-fefd8ae2]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.

Outputs:

vpc_id = vpc-fefd8ae2
vpc_tags = {}
$ terraform apply -auto-approve
aws_vpc.vpc: Refreshing state... [id=vpc-fefd8ae2]
aws_vpc.vpc: Modifying... [id=vpc-fefd8ae2]

Error: error waiting for EC2 VPC (vpc-fefd8ae2) IPv6 CIDR to become disassociated: couldn't find resource (2 retries)


$ terraform apply -auto-approve
aws_vpc.vpc: Refreshing state... [id=vpc-fefd8ae2]
aws_vpc.vpc: Modifying... [id=vpc-fefd8ae2]
aws_vpc.vpc: Modifications complete after 0s [id=vpc-fefd8ae2]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.

Outputs:

vpc_id = vpc-fefd8ae2
vpc_tags = {
  "name" = "some-vpc"
}
```

## Terraform Config Files

main.tf
```
resource "aws_vpc" "vpc" {
  cidr_block                       = "10.0.0.0/16"
  tags = {
    name = "some-vpc"
  }
}
```

site.tf
```
provider "aws" {
  region                      = "us-east-1"
  access_key                  = "mock_access_key"
  secret_key                  = "mock_secret_key"
  s3_force_path_style         = true
  skip_credentials_validation = true
  skip_metadata_api_check     = true
  skip_requesting_account_id  = true
  endpoints {
    ec2            = "http://localhost:4566"
  }
}
```

outputs.tf
```
output "vpc_id" {
  value       = aws_vpc.vpc.id
}

output "vpc_tags" {
  value       = aws_vpc.vpc.tags
}
```

**Comment by whummer** [comment]
Thanks for reporting @smslattery . This should be fixed upstream in spulec/moto#3338 and is now available in the `latest` LocalStack version. Can you please pull the Docker image and give it another try? Please report here if the problem persists. Thanks!

**Comment by smslattery** [comment]
Confirmed as fixed. Tested this against VPC and Subnets with both localstack/localstack:latest and localstack/localstack-full:latest.

Thanks!
